VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -348,16 +362,17 @@ public async Task SendTransferRequest(Guid conferenceId, Guid participantId, Tra
                 var participant = conference.Participants.SingleOrDefault(x => x.Id == participantId);
                 if (participant == null)
                 {
-                    _logger.LogDebug("Participant {participant} does not exist in {conference}", participantId, conferenceId);
+                    _logger.LogDebug("Participant {participant} does not exist in {conference}", participantId,
+                        conferenceId);
                     throw new ParticipantNotFoundException(conferenceId, Context.User.Identity.Name);
                 }
 
                 await Clients.Group(VhOfficersGroupName)
-                    .ParticipantMediaStatusMessage(conferenceId, participantId, mediaStatus);
+                    .ParticipantMediaStatusMessage(participantId, conferenceId, mediaStatus);
```

Comment:
we switching these to confuse people? :P 
others like `HearingTransfer` and `SendTransferRequest` etc are around the original way?

Reply:
As discussed, the variables were called in the wrong order. The signature requires participant id and then conference id. This keeps it similar to the participant status message, but this probably can be refactored so that all messages have conference id first.